From eb73f106729b796baf6445e828a6add6bda3a3ec Mon Sep 17 00:00:00 2001
From: Shunqian Zheng <zhengsq@rock-chips.com>
Date: Fri, 22 Jun 2018 12:00:21 +0800
Subject: [PATCH] ASoC: rk312x: fix duplicate const

SOC_ENUM_SINGLE_DECL already defined with const, removing
the duplicate const in rk312x_codec.c

Change-Id: I085b1faf99a9ccd3b06e4e642abc1bc43cc8bac7
Signed-off-by: Shunqian Zheng <zhengsq@rock-chips.com>
---
 sound/soc/codecs/rk312x_codec.c | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/sound/soc/codecs/rk312x_codec.c b/sound/soc/codecs/rk312x_codec.c
index 64cbd69bed49..e5f459570def 100644
--- a/sound/soc/codecs/rk312x_codec.c
+++ b/sound/soc/codecs/rk312x_codec.c
@@ -616,12 +616,12 @@ static const char *const rk312x_voice_call_path_mode[] = {
 		"OFF", "RCV", "SPK", "HP", "HP_NO_MIC", "BT"};
 
 
-static const SOC_ENUM_SINGLE_DECL(rk312x_playback_path_type, 0, 0,
-				  rk312x_playback_path_mode);
-static const SOC_ENUM_SINGLE_DECL(rk312x_capture_path_type, 0, 0,
-				  rk312x_capture_path_mode);
-static const SOC_ENUM_SINGLE_DECL(rk312x_voice_call_path_type, 0, 0,
-				  rk312x_voice_call_path_mode);
+static SOC_ENUM_SINGLE_DECL(rk312x_playback_path_type, 0, 0,
+			    rk312x_playback_path_mode);
+static SOC_ENUM_SINGLE_DECL(rk312x_capture_path_type, 0, 0,
+			    rk312x_capture_path_mode);
+static SOC_ENUM_SINGLE_DECL(rk312x_voice_call_path_type, 0, 0,
+			    rk312x_voice_call_path_mode);
 
 
 /* static int rk312x_codec_power_up(int type); */
-- 
2.35.3

